Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # Build stage
 #
-FROM maven:3.6.0-jdk-11-slim AS build
+FROM maven:3.8.1-jdk-11-slim AS build
```

Comment:
We changed the version because we have version 3.8.1 in pom and 3.6 in docker and because of this we got error when deploing

Reply:
I missed why you turned off tests? I saw `-Dmaven.test.skip=true` in Dockerfile

Reply:
I only know that docker doesn't start with tests and Lesha is trying to solve this problem

Reply:
keep it in mind please or better create ticket for resolving problem